client/app/scripts/superdesk-authoring/views/send-item.html

```diff
@@ -1,67 +1,73 @@
-<div class="pane-holder" ng-class="{show: isActive}">
+<div class="pane-holder outer-pane" ng-class="{show: isActive}">    
     <div class="send-to-pane">
         <header>
-            <h3 translate ng-show="mode !== 'archive'">Send</h3>
+            <h3 translate ng-show="mode !== 'archive'">Send to / Publish</h3>
             <h3 translate ng-show="mode === 'archive'">Duplicate</h3>
             <div class="close" ng-click="close()"><i class="icon-close-small icon-white"></i></div>
         </header>
         <div class="content">
-            <div class="dropdown desk-select" dropdown>
-                <button class="dropdown-toggle" dropdown-toggle>
-                    <span ng-show="!selectedDesk" translate>Select desk</span>
-                    <span ng-show="selectedDesk">{{selectedDesk.name}}</span>
-                    <b class="caret white"></b>
-                </button>
-                <div class="dropdown-menu">
-                    <ul>
-                        <li ng-repeat="desk in desks._items">
-                            <button ng-click="selectDesk(desk)">{{desk.name}}</button>
-                        </li>
-                    </ul>
+            <div sd-toggle-box data-title="{{ 'Destination' | translate }}" data-open="true">
+                <div class="dropdown desk-select" dropdown>
+                    <button class="dropdown-toggle" dropdown-toggle>
+                        <span ng-show="!selectedDesk" translate>Select desk</span>
+                        <span ng-show="selectedDesk">{{::selectedDesk.name}}</span>
+                        <b class="caret white"></b>
+                    </button>
+                    <div class="dropdown-menu">
+                        <ul>
+                            <li ng-repeat="desk in desks._items track by $index">
```

Comment:
Indeed, because in some edge cases `$index` can even cause quirky behavior. If available, it's best to use a value that uniquely identifies an item and distinguishes it from other items.